### ISSUE TITLE
The HTML `lang` attribute does not have the correct value on non-English pages

### ISSUE THREAD

**Issue by huyenltnguyen** [freeCodeCamp/freeCodeCamp#42727]
**Affected page**
I found the issue on the home page, /learn, and /settings, but I think it exists on all FCC pages as we set language to one of the files in the root of /client.

**To Reproduce**

1. Go to a non-English page. For example: https://chinese.freecodecamp.org/
2. Open the browser console and inspect the DOM

**Actual behavior**
The `lang` attribute of the `html` element is always set to `en`.

**Expected behavior**
The attribute should be the code of the language the page is in. It is to allow screen readers to detect and use the correct pronunciation.

Reference: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang

**Screenshots**

<img width="1241" alt="Screen Shot 2021-07-03 at 07 03 52" src="https://user-images.githubusercontent.com/25715018/124338028-f5979f80-dbcf-11eb-9ef2-c143863dd5fe.png">
